src/Presentation/Nop.Web/Views/Shared/_Root.Head.cshtml

```diff
@@ -11,7 +11,7 @@
 @inject AppSettings appSettings
 
 @{
-
+    NopHtml.AppendScriptParts(ResourceLocation.Footer, "~/lib_npm/swiper/swiper-bundle.min.js");

```

Comment:
Inject `CatalogSettings` and check whether the feature is enabled and we should add a script for it.